frontend/packages/css/package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "@code-dot-org/legacy-css",
+  "name": "@code-dot-org/css",
```

Comment:
If you renamed the package already, let's call it `@code-dot-org/dsco-styles ` and close need of renaming it in the future at once.
`@code-dot-org/css` is nice, but not specific enough :(

Reply:
themes is too specific, this package is not only about themes